src/o365/spfx/commands/project/project-upgrade/upgrade-1.3.1.ts

```diff
@@ -0,0 +1,5 @@
+import { FN010001_YORC_version } from "./rules/FN010001_YORC_version";
+
+module.exports = [
+  new FN010001_YORC_version('1.3.1'),
```

Comment:
@waldekmastykarz  should'd we remind that the packages have to be upgraded as well?  As example, I have the habit to use static versions without ranges so when I create new solution, the first thing to do is to open the packages.json file and make the versions from `"spfx":"~1.3.0"` to `"spfx":"1.3.0"`. Mentioning the version of the packages has changed could help ppl like me :)

Reply:
When packages changed, we do that, but I noticed in the SPFx@1.3.0-1.3.2 range that packages are referenced as `~1.3.0`. If you look at the available SPFx packages there is 1.3.0 and 1.3.4 but no 1.3.1 or 1.3.2. My current reasoning was, that if you installed the project back then, you'd get 1.3.0. If you however, didn't have a lockfile and would install the project today, you'd get 1.3.4, so that's kind of an edge case.

Reply:
Interesting, I did not know about that. Thanks for the clarification!

Reply:
Thanks for double checking. It's always helpful to have a second pair of eyes have a look at the code.